packages/fxa-payments-server/src/components/TermsAndPrivacy/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
```

Comment:
:partyparrot: This helped clean this up for sure. I wonder why we store these links in config, they don't change per ENV do they?

Reply:
These links were using config already. I think this was an ask from higher up. They do not change per ENV